maths/miller_rabin.py

```diff
@@ -0,0 +1,46 @@
+from binary_exp_mod import bin_exp_mod
+import random
+
+# This is a probabilistic check to test primality, useful for big numbers!
+def is_prime(n, prec = 1000):
+  if (n < 2):
```

Comment:
Please add one or more doctests here as discussed in CONTRIBUTING.md.

Reply:
Added a test on it